GitHub: feat(meta): update security policy (#1161)
starlite-api/starlite: SECURITY.md

# Security Policy

Thanks for helping make Starlite safe for everyone.

## Security 

> Starlite takes the security of our projects and services seriously, including all of the open source code repositories managed through our GitHub organizations, such as [@starlite-api](https://github.com/starlite-api).

Reviewer: ```suggestion
Starlite takes the security of our projects and services seriously, including all of the repositories managed by the [starlite-api organization](https://github.com/starlite-api).
```

We will ensure that your finding gets escalated to the appropriate maintainer(s) for remediation. 

## Reporting Security Issues

**Please do not report security vulnerabilities through public GitHub issues, discussions, or pull requests.**

[Click to Open a Security Advisory Privately](https://github.com/starlite-api/starlite/security/advisories/new)

If you believe you have found a security vulnerability in any Starlite-owned repository, please report it to us through coordinated disclosure:
  * In the affected repository, browse to the **Security** tab, select **Advisories**, select "Report a vulnerability"
      * ![image](https://user-images.githubusercontent.com/45884264/217041010-8fd6b96b-329d-4d8e-8838-9b5bf4e1a78d.png)
  * Please Do **NOT** Creating an issue in the affected repository

>   * Please do **NOT** Private message or tag "@maintainer" on our [Discord server](https://discord.gg/MmcwxztmQb)

Reviewer: ```suggestion
  * Please do **NOT** creating an issue in the affected repository
  * Please do **NOT** send a private message to or tag "@maintainer" on our [Discord server](https://discord.gg/MmcwxztmQb)
```

Author: (meta) annoying github feature: you cant commit suggestions to the same file even if they dont affect the same line 
![image](https://user-images.githubusercontent.com/45884264/217050148-7de851af-c9d6-4141-aa09-567cbfbbd8dc.png)

maybe that's what the batch feature is for? :(

will get this fixed
